.github/README.md

```diff
@@ -37,12 +38,25 @@ Looking for the most stable or the latest avaliable download? Both the latest st
 * All core IO functionality extracted into a library for use in external tools.
 * Can generate an .yydebug file for the GM:S debugger so that you can edit variables live! (see [here](https://github.com/krzys-h/UndertaleModTool/wiki/Corrections-to-GameMaker-Studio-1.4-data.win-format-and-VM-bytecode,-.yydebug-format-and-debugger-instructions#yydebug-file-format))
 
+# Screenshots
+
+Here's some screenshots of what UTMT can do:
+
+## [AM2R: Horde Mode Mod](https://github.com/Hornoads/AM2R-The-Horde-Multitroid/releases)
+<img src="images/am2r.png" alt="AM2R: Horde Mode"/>
+
+## [Undertale: Bits and Pieces Mod](https://gamejolt.com/games/UndertaleBnP/574044)
+<img src="images/bnp-ut.png" alt="BnP"/>
+
+## [RIBBIT - The Deltarune Mod](https://gamejolt.com/games/ribbitmod/671888)
+<img src="images/ribbit-dr.png" alt="RIBBIT"/>
```

Comment:
Can you make these images have a consistent size please?

Reply:
that would lead to some image artifacts

Reply:
Wouldn't be very noticeable, due to the small size anyway. I just don't feel it looks good if
a) all images have varying sizes
b) the readme is cluttered with huge images.